cgo_flags.go

```diff
@@ -3,7 +3,7 @@
 package jemalloc
 
 // #cgo CFLAGS: -Iinternal/include
-// #cgo linux CFLAGS: -DJEMALLOC_PURGE_MADVISE_DONTNEED
-// #cgo darwin CFLAGS: -DJEMALLOC_OSATOMIC -DJEMALLOC_OSSPIN -DJEMALLOC_PURGE_MADVISE_FREE -DJEMALLOC_HAVE_ISSETUGID
-// #cgo darwin CFLAGS: -DJEMALLOC_ZONE -DJEMALLOC_ZONE_VERSION=8
+// #cgo darwin CFLAGS: -Idarwin_includes -Idarwin_includes/jemalloc/internal
+// #cgo linux CFLAGS: -Ilinux_includes -Ilinux_includes/jemalloc/internal
+// #cgo linux LDFLAGS: -lm
```

Comment:
On OSX, both `libc` and `libm` are provided by `libSystem` which is linked by default (http://stackoverflow.com/questions/33675638/gcc-link-the-math-library-by-default-in-c-on-mac-os-x)

But it's still a little more correct to pass `-lm` even on darwin so we should just remove the platform tag from this line.